scaling.adoc

```diff
@@ -235,37 +234,36 @@ different versions of Bitcoin Core going back to version 0.8.
 image::Bitcoin-Core-Sync-Performance-1.png[{big-width}]
 
-The different lines represents different versions of Bitcoin Core. The
-leftmost line is the latest, as of writing, version 22.0 released
-September 2021 which took 396 minutes to fully sync, and the rightmost
-one is version 0.8 from November 2013 which took 3452 minutes. All of
-this, roughly 10x, improvement is due to inward scaling.
+The different lines represent different versions of Bitcoin Core. The leftmost
+line is the latest; version 22.0, released in September 2021, took 396 minutes
+to fully sync. The rightmost one is version 0.8 from November 2013, which took
+3452 minutes. All of this, roughly 10x, improvement is due to inward scaling.
 
-The improvements made could be categorized as either space (RAM, disk,
-bandwidth, etc) savings or computational savings. Both categories
+The improvements could be categorized as either space (RAM, disk,
+bandwidth, etc.) savings or computational savings. Both categories
 contribute to the improvements in the diagram above.
 
 A good example of computational improvements can be found in the
 https://github.com/bitcoin-core/secp256k1[libsecp256k1] library, which
-among other things implements the cryptographic primitives needed to
+among other things, implements the cryptographic primitives needed to
 make and verify digital signatures. Pieter Wuille is one of the
-contributors to this library and he
+contributors to this library, and he
 https://twitter.com/pwuille/status/1450471673321381896[wrote a twitter
 thread] showcasing the performance improvements made by various pull
 requests.
 
 .Performance of signature verification over time, with significant pull requests marked on the timeline. Source: https://twitter.com/pwuille/status/1450471673321381896
 image::libsecp256k1speedups.png[{half-width}]
 
-The graph shows the trend for two different 64-bit CPU types, ARM and
-x86. The difference in performance is due to the more specialized
-instructions available on x86, compared to the ARM architecture which
-has fewer more generic instructions. But the general trend is the same
-for both architectures. Note that the Y-axis is logarithmic, which
-makes the improvements look less impressive than they actually are.
+The graph shows the trend for two different 64-bit CPU types, ARM and x86.
+The difference in performance is due to the more specialized instructions
+available on x86 compared to the ARM architecture, which has fewer,
+more generic instructions. But the general trend is the same for both
+architectures. Note that the Y-axis is logarithmic, which makes the
+improvements look less impressive than they actually are.
 
-There are also several good examples of where space savings have
-contributed to performance improvements. In a
+TThere are also several good examples of space savings contributing to
```

Comment:
TT

Reply:
thanks, addressed in https://github.com/kallerosenbaum/btcphilosophy/commit/0e8d06c4d600577181c6e1cd933bfc50db412809